raster-foundry/raster-foundry
# Remove Scapegoat errors for common project

## Overview

Remedy the warning messages and use `SuppressWarnings` where a straightforward solution isn't obvious. Also, clean up imports across all files touched.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~


~- [ ] Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
~- [ ] Any new SQL strings have tests~

## Testing Instructions

Execute the following command and confirm the Scapegoat output:

```bash
$ docker-compose run --rm --no-deps api-server common/scapegoat

...

[info] [info] [scapegoat] 117 activated inspections
[info] [info] [scapegoat] List(.*/datamodel/.*) ignored file patterns
[info] [info] [scapegoat] Analysis complete: 26 files - 0 errors 0 warns 0 infos
[info] [info] [scapegoat] Written HTML report [/opt/raster-foundry/app-backend/common/target/scala-2.11/scapegoat-report/scapegoat.html]
[info] [info] [scapegoat] Written XML report [/opt/raster-foundry/app-backend/common/target/scala-2.11/scapegoat-report/scapegoat.xml]
[info] [info] [scapegoat] Written Scalastyle XML report [/opt/raster-foundry/app-backend/common/target/scala-2.11/scapegoat-report/scapegoat-scalastyle.xml]
...
```

Connects https://github.com/raster-foundry/raster-foundry/issues/3758
